spec/fixtures/multiple_bibs.json

```diff
@@ -70,7 +70,7 @@
edia -- Ethics.</subfield></datafield><datafield ind1=\" \" ind2=\"0\" tag=\"650\"><subfield code=\"a\">Journalism</subfield><subfield code=\"v\">Handbooks, manuals, etc.</subfield></datafield><datafield ind1=\" \" ind2=\"0\" tag=\"650\"><subfield code=\"a\">Journalism</subfield><subfield code=\"x\">Authorship</subfield><subfield code=\"v\">Handbooks, manuals, etc.</subfield></datafield><datafield ind1=\" \" ind2=\"0\" tag=\"650\"><subfield code=\"a\">Broadcast journalism</subfield><subfield code=\"x\">Authorship</subfield><subfield code=\"v\">Handbooks, manuals, etc.</subfield></datafield><datafield ind1=\" \" ind2=\"0\" tag=\"650\"><subfield code=\"a\">Online journalism</subfield><subfield code=\"x\">Authorship</subfield><subfield code=\"v\">Handbooks, manuals, etc.</subfield></datafield><datafield ind1=\" \" ind2=\"1\" tag=\"945\"><subfield code=\"i\">39074026732875</subfield><subfield code=\"k\">MAIN</subfield><subfield code=\"l\">stacks</subfield><subfield code=\"a\">PN4775</subfield><subfield code=\"b\">.F45 2019</subfield><subfield code=\"z\">090</subfield></datafield><datafield ind1=\" \" ind2=\" \" tag=\"960\"><subfield code=\"k\">MAIN</subfield><subfield code=\"l\">stacks</subfield></datafield><datafield ind1=\" \" ind2=\" \" tag=\"979\"><subfield code=\"a\">(OCoLC)935193850</subfield></datafield><datafield ind1=\" \" ind2=\" \" tag=\"980\"><subfield code=\"a\">02-07-18</subfield><subfield code=\"b\">9500</subfield><subfield code=\"e\">7647</subfield><subfield code=\"f\">I20108006</subfield></datafield><datafield ind1=\" \" ind2=\" \" tag=\"981\"><subfield code=\"a\">Coutt</subfield><subfield code=\"b\">ctssa</subfield><subfield code=\"d\">33587890</subfield><subfield code=\"e\">SLI</subfield><subfield code=\"k\">MAIN</subfield><subfield code=\"l\">stacks</subfield></datafield><datafield ind1=\" \" ind2=\" \" tag=\"997\"><subfield code=\"a\">20180220</subfield><subfield code=\"d\">TEU</subfield><subfield code=\"c\">COUTTS SR</subfield><subfield code=\"e\">MARCIVE</subfield><subfield code=\"e\">WCAT</subfield></datafield><datafield ind1=\" \" ind2=\" \" tag=\"INT\"><subfield code=\"a\">P</subfield></datafield><datafield ind1=\" \" ind2=\" \" tag=\"INST\"><subfield code=\"a\">01TULI_INST</subfield></datafield><datafield ind1=\" \" ind2=\" \" tag=\"AVA\"><subfield code=\"0\">991036880906903811</subfield><subfield code=\"8\">22413618700003811</subfield><subfield code=\"a\">01TULI_INST</subfield><subfield code=\"b\">MAIN</subfield><subfield code=\"c\">Stacks</subfield><subfield code=\"d\">PN4775 .F45 2019</subfield><subfield code=\"e\">available</subfield><subfield code=\"f\">1</subfield><subfield code=\"g\">0</subfield><subfield code=\"i\">m</subfield><subfield code=\"j\">stacks</subfield><subfield code=\"k\">0</subfield><subfield code=\"p\">1</subfield><subfield code=\"q\">Paley Library</subfield><subfield code=\"t\">holding-1</subfield><subfield code=\"t\">holding-2</subfield><subfield code=\"t\">holding-3</subfield></datafield></record>"
```

Comment:
Are these `nill` subfield values supposed to evaluate to a `nil` value when parsed?

Reply:
I haven't run this code, but I suspect another approach is needed to ensure that actual `nil` values are properly handled.

Reply:
Basically two questions: if the code is empty, or not defined, shall it be ignored? if the content is empty, shall it be ignored?

Reply:
[record-991853173503681.json](https://github.com/user-attachments/files/17805802/record-991853173503681.json)

Reply:
This is the record the causes problem: there are two subfields like this: {"code"=>"t"}, {"code"=>"t"}